Library/Formula/bazel.rb

```diff
@@ -0,0 +1,23 @@
+class Bazel < Formula
+  desc "Google's own build tool"
+  homepage "http://bazel.io/"
+  url "https://github.com/bazelbuild/bazel/releases/download/0.1.1/bazel-0.1.1-installer-darwin-x86_64.sh"
+  version "0.1.1"
+  sha256 "2d34e19a47067f8ab793c634146a117552666120c690630d0c9ab0039aafb48a"
+
+  bottle :unneeded
+
+  depends_on :java
+
+  def install
+    chmod 0755, "./bazel-0.1.1-installer-darwin-x86_64.sh"
+    system "./bazel-0.1.1-installer-darwin-x86_64.sh", "--prefix=#{libexec}",
+                                                       "--bazelrc=#{etc}"
+
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+  end
+
+  test do
+    system "#{bin}/bazel", "version"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!